trade-mapping-to-ipaffs/src/test/resources/chedp/chedp_ehc_complete.json

```diff
@@ -419,6 +424,24 @@
         "subjectCode": {
           "value": "CONFORMS_TO_EU"
         }
+      },
+      {
+        "content": [
+          {
+          "value": "TESTDOCUMENT"
+          }
+        ],
+        "subjectCode": {
+          "value": "transport_to_bcp_document"
+        }
+      },
+      {
+        "content" : [ {
+          "value" : "2020-06-15T21:30:00Z"
+        } ],
+        "subjectCode" : {
+          "value" : "DEPARTURE_FROM_BCP_DATETIME"
```

Comment:
What's this note for?

Reply:
its for the departure time in the integration test

Reply:
The [docs](https://dev.azure.com/defragovuk/STW%20Integration/_wiki/wikis/STW-Integration.wiki/23309/CHEDP-Payload-Description) say that departure date should come from the `spsConsignment.exportExitDateTime` but I think I like this better - we should update the payload description pages to reflect this

Reply:
I've updated the docs